app/assets/stylesheets/screens/index.scss

```diff
@@ -1 +1,2 @@
-@import './keywords'
+@import './keywords';
+@import './keyword_details';
```

Comment:
As we have the controller as the action on the body tag (`<body class="<%= "#{controller_name.dasherize} #{action_name.dasherize}" %> px-4 py-2">`), we no need to separate it into 2 files for the screen @Goose97, instead we can have only 1 `screens/_keywords.scss` and the content for that separate per action

```
body.keywords {
  &.index {
    .....
  }

  &.show {
   ....
  }
```

Reply:
Good call. Fixed in 9e620782ea3de6556291ff7e0a21faf4485716f2